### PR TITLE
v1.6: Add token program hack

### PR DESCRIPTION
**Pull request by joncinque**
Hack at the level of the BPF loader instead of the account db load.  It doesn't have the neatness of using the exact account from the db, but `include_bytes!` makes this still pretty easy.

### CODE REVIEW

File: upload-perf/src/upload-perf.rs [hunk 1]
```diff
@@ -74,7 +74,6 @@ fn main() {
                         ("commit", git_commit_hash.trim().to_string(), String)
                     );
                     */
-
```
Comment (mvines) [inline upload-perf/src/upload-perf.rs:77]: 🔍  What is this stray whitespace!

**Review by mvines — COMMENTED** [review]
looks like it should work!

File: upload-perf/src/upload-perf.rs (hunk 1 above)
Comment (joncinque) [reply upload-perf/src/upload-perf.rs:77]: :weary: this is from the new version of `cargo fmt` -- the old version forces this whitespace (not the first time this has happened to me)

Comment (mvines) [reply upload-perf/src/upload-perf.rs:77]: r-

**Pull request closed without merging**